mobile/src/main/java/ch/epfl/sdp/mobile/ui/play/PlayScreen.kt

```diff
@@ -0,0 +1,52 @@
+package ch.epfl.sdp.mobile.ui.play
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material.ExtendedFloatingActionButton
+import androidx.compose.material.Icon
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import ch.epfl.sdp.mobile.state.LocalLocalizedStrings
+import ch.epfl.sdp.mobile.ui.Add
+import ch.epfl.sdp.mobile.ui.PawniesColors
+import ch.epfl.sdp.mobile.ui.PawniesIcons
+
+/**
+ * Composable that composes the PlayScreen [TODO] Contains a new game button only, should be
+ * expanded to include history of matches
+ *
+ * @param state state of the PlayScreen
+ * @param modifier the [Modifier] for this composable.
+ */
+@Composable
+fun PlayScreen(
+    state: PlayScreenState,
+    modifier: Modifier = Modifier,
+    contentPadding: PaddingValues = PaddingValues(),
+) {
+  Box(
+      contentAlignment = Alignment.BottomEnd,
+      modifier = modifier.fillMaxSize().padding(contentPadding)) {
+    NewGameButton(onNewGame = state.onNewGameClick)
```

Comment:
Prefer aligning the individual Button component rather than all the children of the `Box`.

```suggestion
  Box(modifier = modifier.fillMaxSize().padding(contentPadding)) {
    NewGameButton(onNewGame = state.onNewGameClick, modifier = Modifier.align(Alignment.BottomEnd))
```